### PR TITLE
Updates antagonist Dummy's

### PR DESCRIPTION
**Pull request by gonenoculer5**
Updates the antagonist datums to have actual nice looking dummies.

## About The Pull Request

Antag preview icons didn't have update_hair() set to TRUE, making none of the hair changes show. This has been remidied.

## Changelog

:cl: Gonenoculer5
fix: We've given the antags a good spray of Barbers Aid, so they all have proper hair now. Someone should shoot the person that gave them all baldium.
/:cl:

### CODE REVIEW

File: code/modules/antagonists/traitor/datum_traitor.dm [hunk 1]
```diff
@@ -349,3 +349,16 @@
 	sword.worn_icon_state = "e_sword_on_red"
 
 	H.update_inv_hands()
+
+
+/datum/antagonist/traitor/get_preview_icon()
+	var/mob/living/carbon/human/dummy/consistent/traitor_dummy = new
+	traitor_dummy.hair_color = "#dba86e53"
+	traitor_dummy.hairstyle = "CIA"
+	traitor_dummy.update_hair(TRUE)
```
Comment (SabreML) [inline code/modules/antagonists/traitor/datum_traitor.dm:358]: I'm not sure about the 'CIA' hairstyle on this one. Combined with the gas mask it looks a bit like some kind of weird hat:
image

File: code/modules/antagonists/creep/creep.dm [hunk 2]
```diff
@@ -37,12 +37,12 @@
...
-	victim_dummy.hair_color = "#bb9966" // Brown
-	victim_dummy.hairstyle = "Messy"
-	victim_dummy.update_hair()
+	var/mob/living/carbon/human/dummy/consistent/obsessed_dummy = new
+	obsessed_dummy.hair_color = "#bb9966" // Brown
+	obsessed_dummy.hairstyle = "Messy"
+	obsessed_dummy.update_hair(TRUE)
 
-	var/icon/obsessed_icon = render_preview_outfit(preview_outfit)
+	var/icon/obsessed_icon = render_preview_outfit(preview_outfit,obsessed_dummy)
```
Comment (SabreML) [inline code/modules/antagonists/creep/creep.dm:45]: Nitpicks:
```suggestion
	var/icon/obsessed_icon = render_preview_outfit(preview_outfit, obsessed_dummy)
```

File: code/modules/antagonists/traitor/datum_traitor.dm [hunk 3]
```diff
@@ -349,3 +349,16 @@
...
+
+	var/icon/traitor_icon = render_preview_outfit(preview_outfit,traitor_dummy)
```
Comment (SabreML) [inline code/modules/antagonists/traitor/datum_traitor.dm:360]:
```suggestion
	var/icon/traitor_icon = render_preview_outfit(preview_outfit, traitor_dummy)
```

File: code/modules/antagonists/traitor/datum_traitor.dm (hunk 1 above)
Comment (gonenoculer5) [reply code/modules/antagonists/traitor/datum_traitor.dm]: Changed to business hair and made grey-ish black.

Comment (gonenoculer5) [reply code/modules/antagonists/traitor/datum_traitor.dm]: image

**Review by SabreML — APPROVED** [review]

**Pull request merged**